independent-publisher-connectors/SOS Inventory/apiDefinition.swagger.json

```diff
@@ -0,0 +1,2218 @@
+														"description": "Country"
+													},
+													"phone": {
+														"type": "string",
+														"description": "Phone"
+													},
+													"email": {
+														"type": "string",
+														"description": "Email"
+													},
+													"company": {
+														"type": "string",
+														"description": "Company"
+													},
+													"contact": {
+														"type": "string",
+														"description": "Contact"
+													}
+												}
+											}
+										}
+									}
+								}
+							}
+						}
+					}
+				},
+				"summary": "Locations",
+				"description": "List Locations",
+				"operationId": "Location",
+				"x-ms-visibility": "important",
+				"parameters": [{
+					"name": "limit",
+					"in": "query",
+					"x-ms-summary": "Limit",
+					"description": "Limit number of locations returned. Default 10",
+					"type": "integer",
+					"default": 10
+				}]
+			}
+		},
+		"/customer/{id}": {
+			"get": {
+				"responses": {
+					"200": {
+						"description": "OK",
+						"schema": {
+							"$ref": "#/definitions/customer"
+						}
+					}
+				},
+				"summary": "Get a Customer",
+				"description": "Gets a Customer by ID",
+				"operationId": "GetCustomer",
+				"x-ms-visibility": "important",
+				"parameters": [{
+					"name": "id",
+					"required": true,
+					"in": "path",
+					"x-ms-summary": "id",
+					"description": "Customer ID in SOS Inventory.",
+					"type": "number",
+					"x-ms-visibility": "important",
+					"x-ms-url-encoding": "single"
+				}]
+			}
+		},
+		"/salesorder/{id}": {
+			"get": {
+				"responses": {
+					"200": {
+						"description": "OK",
+						"schema": {
+							"$ref": "#/definitions/salesorder"
+						}
+					}
+				},
+				"summary": "Get a Sales Order",
+				"description": "Gets a Sales Order by ID",
+				"operationId": "GetSalesOrder",
+				"x-ms-visibility": "important",
+				"parameters": [{
+					"name": "id",
+					"required": true,
+					"in": "path",
+					"x-ms-summary": "id",
+					"description": "Order ID in SOS Inventory.",
+					"type": "number",
+					"x-ms-visibility": "important",
+					"x-ms-url-encoding": "single"
+				}]
+			}
+		},
+		"/salesorder": {
+			"get": {
+				"responses": {
+					"200": {
+						"description": "OK",
+						"schema": {
+							"type": "object",
+							"properties": {
+								"count": {
+									"type": "integer",
+									"description": "# Sales Orders Returned"
+								},
+								"totalCount": {
+									"type": "integer",
+									"description": "# Matching Sales Orders"
+								},
+								"data": {
+									"type": "array",
+									"items": {
+										"$ref": "#/definitions/salesorder"
+									}
+								}
+							}
+						}
+					}
+				},
+				"summary": "List Sales Orders",
+				"description": "List Sales Orders",
+				"operationId": "SalesOrder",
+				"x-ms-visibility": "important",
+				"parameters": [{
```

Comment:
@Harold-Anderson please make sure that the formatting is correct. If it's an array of objects, the curly brace should be on the next line.